### PR TITLE
format header

### PR DESCRIPTION
**Pull request by rccsousa**
Why:
- The current implementation of the Content Hub is striped of styling. This PR starts by styling the header.

How:
- added dropdown menus and styling;
- navbar feeds on the header links provided on CMS;
- updated globals.css with some minor changes;

### CODE REVIEW

File: src/app/_css/globals.css [hunk 1]
```diff
@@ -26,6 +25,12 @@
   --dark-rock-800: #403F4C;
   --soft-white-100: #F6F6F6;
 
+  /* Links */
+  a {
```
Comment (davelange) [inline src/app/_css/globals.css:29]: Btw, there are some default styling behaviours of browsers which are a pain. A good idea is to use a "CSS reset" (like this one https://www.joshwcomeau.com/css/custom-css-reset/), it helps to have more of a blank slate to work with.

File: src/app/_components/Header/index.tsx [hunk 2]
```diff
@@ -1,35 +1,48 @@
...
+      <header className={styles.container}>
+        <div>
+          <Logo/>
+        </div>
+
+        {/* TODO: Conditionally format Content Hub to reflect active link?*/}
+        <nav className={styles.navbar}>
+          {navItems.map(({ link }, i) => {
+            return (
+              <Link key={i} href={link.url}>
```
Comment (davelange) [inline src/app/_components/Header/index.tsx:35]: Not an issue, but you can skip the "return", and just have

```
 {navItems.map(({ link }, i) => (
     <Link key={i} href={link.url}>
        {link.label}
      </Link>
)}
```
When there are many maps, it's a nice saving

**Review by davelange — APPROVED** [review]
We're missing fonts right? Do you have access to the files?

File: src/app/_css/globals.css (hunk 1 above)
Comment (rccsousa) [reply src/app/_css/globals.css:29]: this is excelent! Thanls for sharing

File: src/app/_components/Header/index.tsx (hunk 2 above)
Comment (rccsousa) [reply src/app/_components/Header/index.tsx:35]: noted. that makes sense, I thought it was necessary given `{ link }` being declared as an object.
thanks

**Comment by rccsousa** [comment]
> We're missing fonts right? Do you have access to the files?

I've declared them but have only some installed locally, don't have access to all of them. I'll get in touch regarding that.

**Pull request merged**